File: exercicios/Python/1929/1929.py
n = input().split()

a = int(n[0])
b = int(n[1])
c = int(n[2])
d = int(n[3])

if 1 <= a and a <= 100 and 1 <= b and b <= 100 and 1 <= c and c <= 100 and 1 <= d and d <= 100: 
    if a + b > c and b + c > a and a + c > b:
        print('S')
    elif b + c > d and c + d > b and b + d > c: 
        print('S')
    elif a + c > d and c + d >a and a + d > c:
        print('S')
    elif a + b> d and b + d > a and a + d > b:
        print('S')
    else:
        print('N')

File: exercicios/JavaScript/1929/1929.js
var input = require('fs').readFileSync('stdin','utf8')
var lines = input.split('\n')

var valores = lines.shift().split(' ')

var a = parseInt(valores[0])
var b = parseInt(valores[1])
var c = parseInt(valores[2])
var d = parseInt(valores[3])

if(1 <= a && a <= 100 && 1 <= b && b <= 100 && 1 <= c && c <= 100 && 1 <= d && d <= 100){
    if(a + b > c && b + c > a && a + c > b){
        console.log('S')}
    else if(b + c > d && c + d > b && b + d > c){ 
        console.log('S')}
    else if(a + c > d && c + d >a && a + d > c){
        console.log('S')}
    else if(a + b> d && b + d > a && a + d > b){
        console.log('S')}
    else{
        console.log('N')}
    }